opal/managers.py

```diff
@@ -68,21 +66,24 @@ def serialised(self, user, episodes,
         If HISTORIC_TAGS is Truthy, return deleted tags as well.
         If EPISODE_HISTORY is Truthy return historic episodes as well.
         """
+        from opal.models import Tagging
+        from opal.models import Patient
+
         patient_ids = [e.patient_id for e in episodes]
+        patients = Patient.objects.filter(id__in=patient_ids)
         patient_subs = defaultdict(lambda: defaultdict(list))
 
         episode_subs = self.serialised_episode_subrecords(episodes, user)
         for model in patient_subrecords():
             name = model.get_api_name()
-            subrecords = model.objects.filter(patient__in=patient_ids)
+            subrecords = model.objects.for_patients(patients)
```

Comment:
Doesn't this mean we do an extra query (above) compared to adding prefetch related here ?

Reply:
possibly, I thought for_patients, was a nicer api than for_patient_ids